lib/roo_on_rails/default.env

```diff
@@ -9,3 +9,8 @@ NEW_RELIC_DEVELOPER_MODE=false
 NEW_RELIC_LICENSE_KEY=override-me
 RACK_SERVICE_TIMEOUT=10
 RACK_WAIT_TIMEOUT=30
+SIDEKIQ_ENABLED=true
+SIDEKIQ_THREADS=5
+SIDEKIQ_DB_POOL_SIZE=6
+SIDEKIQ_DB_REAPING_FREQUENCY=10
```

Comment:
We're actually running at 40 threads because of all the IO; it reduced our dyno count by around 50% compared to the previous week. So I'm not sure there is a sensible default. Was just suggesting that if you don't set anything at all Sidekiq will use 25 and I expect most people would assume that's the case.